### PR TITLE
Fix app crashing

### PR DESCRIPTION
**Pull request by theRoughCode**
- Reformatted file names to be in PascalCase according to [AirBnB guidelines](https://github.com/airbnb/javascript/tree/master/react#naming)
- Lifted single components out of their own folder (unnecessary to have a folder just for one component, each with its own `index.js` file which just points to that component anyway)
- Added `default` to `export default...` so we don't have to import with braces (more readable and clear it is a single component)

### CODE REVIEW

File: client/app/components/faq/faqdetails.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ import {
 import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
 import Icon from 'react-native-vector-icons/Entypo'
 import { TabNavigator, StackNavigator, HeaderBackButton } from 'react-navigation'
-import { FaqStyles } from '../../styles/faqstyles'
-import { Header } from '../header'
-import { ListSlider } from '../listslider'
+import { FaqStyles } from '../../styles/FaqStyles'
```
Comment (amyqiu) [inline client/app/components/faq/faqdetails.js]: I got errors here too, looks to be the same problem as the capitalizing of container.js above

File: client/app/components/faq/faqlist.js [hunk 2]
```diff
@@ -13,11 +13,11 @@ import { ListItem, SearchBar } from 'react-native-elements';
 import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation';
 import Icon from 'react-native-vector-icons/Entypo';
 import { TabNavigator, StackNavigator, HeaderBackButton } from 'react-navigation';
-import { FaqStyles } from '../../styles/faqstyles';
+import { FaqStyles } from '../../styles/FaqStyles';
```
Comment (amyqiu) [inline client/app/components/faq/faqlist.js]: Same problem as with above with the FaqStyles.js spelling

File: client/app/screens/FaqScreen.js [hunk 3]
```diff
@@ -2,8 +2,9 @@ import React, { Component } from 'react';
 import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
 import { TabNavigator, StackNavigator } from 'react-navigation'
 
-import { Header } from '../components/header'
-import { FaqList, FaqDetails } from '../components/faq'
+import Header from '../components/Header'
+import FaqList from '../components/faq/FaqList'
```
Comment (amyqiu) [inline client/app/screens/FaqScreen.js]: Same issue with capitalization of FaqList and FaqDetails

File: client/App.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
 import logger from 'redux-logger';
-import Container from './app/container';
+import Container from './app/Container';
```
Comment (amyqiu) [inline client/App.js]: Got an error here because the packager couldn't find a Container.js file, was container.js was supposed to be renamed?

Comment (theRoughCode) [reply client/App.js]: Yup, the `container.js` file should have been renamed to `Container.js`

File: client/app/components/faq/faqdetails.js (hunk 1 above)
Comment (theRoughCode) [reply client/app/components/faq/faqdetails.js]: The files should have been renamed

File: client/app/screens/FaqScreen.js (hunk 3 above)
Comment (theRoughCode) [reply client/app/screens/FaqScreen.js]: Oops, for some reason, I forgot to capitalize these files.  Thanks Amy!

File: client/app/components/ListSlider.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Swiper from 'react-native-swiper';
 
-export default function ListSlider({ currentIndex, itemList, renderItem }) {
+export default ListSlider = ({ currentIndex, itemList, renderItem }) => {
```
[the remark this answers is not in the document]
Comment (theRoughCode) [reply client/app/components/ListSlider.js]: I prefer using the arrow function over calling it a function.  Personal preference, which I chose to adopt after reading Dan Abramov's [article](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) and the related [example](https://gist.github.com/chantastic/fc9e3853464dffdb1e3c)

File: client/app/components/faq/faqdetails.js [hunk 6]
```diff
@@ -1,28 +1,41 @@
 import React from 'react';
 import {
+	AppRegistry,
 	Text,
 	View,
 } from 'react-native';
-import { FaqStyles } from '../../styles/faqstyles'
-import { Header } from '../header'
-import { ListSlider } from '../listslider'
+import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
```
[the remark this answers is not in the document]
Comment (theRoughCode) [reply client/app/components/faq/faqdetails.js]: Yea you're right, your changes didn't get merged in due to class name change.  I've run `npm run lint` and fixed the mistakes.  Thanks!

File: client/app/components/faq/faqdetails.js [hunk 7]
```diff
@@ -39,12 +37,12 @@ class FaqDetails extends React.Component {
 		const { state } = this.props.navigation;
 		return (
 			<ListSlider
-				renderItem={this.renderItem}
+				renderItem={this._renderItem}
```
Comment (AbhiPrasad) [inline client/app/components/faq/faqdetails.js]: Probably should not be changed as fnc above still called `renderItem`

File: client/app/components/faq/faqdetails.js [hunk 8]
```diff
@@ -3,19 +3,17 @@ import {
...
 class FaqDetails extends React.Component {
 	static navigationOptions = ({ navigation }) => ({
-		header: (
-			<Header
-				title="Information"
-				hasBackButton
-				navigation={navigation}
-			/>
-		)
+		header: <Header
```
Comment (AbhiPrasad) [inline client/app/components/faq/faqdetails.js]: What do you think about keep vs removing the parenthesis here?

File: client/app/components/faq/faqlist.js [hunk 9]
```diff
@@ -59,26 +60,21 @@ class FaqList extends React.Component {
...
-				size={30}
-				color="#787878"
-				style={{ marginTop: 5 }}
-			/>
-		);
-
 		return (
 			<ListItem
-				containerStyle={{ ...FaqStyles.faqListItem, rowBg }}
+				containerStyle={FaqStyles.faqListItem, rowBg}
```
Comment (AbhiPrasad) [inline client/app/components/faq/faqlist.js]: 👍

File: client/app/components/faq/faqlist.js [hunk 10]
```diff
@@ -59,26 +60,21 @@ class FaqList extends React.Component {
...
 				titleStyle={FaqStyles.faqListItemText}
 				key={item.id}
 				title={item.question}
-				onPress={this.renderFaqDetails(item, index)}
-				rightIcon={icon}
+				onPress={() => this._renderFaqDetails(item, index)}
```
Comment (AbhiPrasad) [inline client/app/components/faq/faqlist.js]: Why the change?

File: client/app/components/faq/faqdetails.js (hunk 8 above)
Comment (theRoughCode) [reply client/app/components/faq/faqdetails.js]: Add parenthesis around it?  I think we should add them in.  Not sure why it was removed.  Thanks for the catch!

File: client/app/components/faq/faqdetails.js (hunk 7 above)
Comment (theRoughCode) [reply client/app/components/faq/faqdetails.js]: Error due to merge conflict overwriting past changes.  Fixed, thanks.

File: client/app/components/faq/faqlist.js (hunk 10 above)
Comment (theRoughCode) [reply client/app/components/faq/faqlist.js]: Overwriting due to merge conflicts again.
